src/test/groovy/graphql/execution/ExecutionParametersTest.java

```diff
@@ -0,0 +1,41 @@
+package graphql.execution;
+
+import graphql.language.Field;
+import org.junit.Test;
+
+import java.util.List;
+import java.util.Map;
+
+import static graphql.Scalars.*;
+import static graphql.execution.ExecutionParameters.newParameters;
+import static graphql.execution.TypeInfo.newTypeInfo;
+import static java.util.Collections.emptyList;
+import static java.util.Collections.singletonMap;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotSame;
+import static org.junit.Assert.assertSame;
+
+public class ExecutionParametersTest {
```

Comment:
Java test because of `transform()` lambda. Groovy (until 3.0) doesn't work properly with lambdas and functional interfaces.